pkg/edgeview/README.md

```diff
@@ -11,28 +11,31 @@ It contains an example of golang source for 'edgeview' websocket dispatcher runn
 ## Dockerfiles
```

Comment:
NIT
"edgeview: Fix image name to be aligned" -> "edgeview docs: Fix image name to be consistent"